sharedotlp/pkg/sharedotlptraces/traces_test.go

```diff
@@ -27,7 +27,7 @@ func TestTracesModule(t *testing.T) {
 			config: ModuleConfig{
 				Exporter: OTLPExporter,
 				OTLPConfig: &OTLPConfig{
-					Mode:     sharedotlp.ModeGRPC,
+					Mode:     pkg.ModeGRPC,
```

Comment:
maybe it's better not to use "pkg" as package name, might be problematic later on